build-tools/packages/build-cli/src/args.ts

```diff
@@ -39,6 +44,26 @@ export const findPackageOrReleaseGroup = (
 	);
 };
 
+/**
+ * Creates a CLI argument for release group names. It's a factory function so that commands can override the
+ * properties more easily when using the argument.
+ */
+export const releaseGroupArg = Args.custom({
+	name: "release_group",
+	required: true,
+	description: "The name of a release group.",
+});
+
+/**
+ * Takes a release group name and searches the build project for it.
+ */
+export const findReleaseGroup = (
+	name: string,
+	buildProject: IBuildProject,
+): IReleaseGroup | undefined => {
+	return buildProject.releaseGroups.get(name as ReleaseGroupName);
+};
```

Comment:
After our chat and I see how simple this function is, I'm not sure it's any better than just calling `buildProject.releaseGroups.get(name as ReleaseGroupName);` directly. That said, it's fine to keep it.